lib/ansible/module_utils/cloud.py

```diff
@@ -106,3 +159,62 @@ def retry_func(*args, **kwargs):
             return retry_func  # true decorator
 
         return deco
+
+    @classmethod
+    def exponential_backoff(cls, retries=10, delay=3, backoff=2, max_delay=60):
+        """
+        Retry calling the Cloud decorated function using an exponential backoff.
+
+        Kwargs:
+            retries (int): Number of times to retry a failed request before giving up
+                default=10
+            delay (int or float): Initial delay between retries in seconds
+                default=3
+            backoff (int or float): backoff multiplier e.g. value of 2 will
+                double the delay each retry
+                default=1.1
+            max_delay (int or None): maximum amount of time to wait between retries.
+                default=60
+        """
+        return cls._backoff(_exponential_backoff(
+            retries=retries, delay=delay, backoff=backoff, max_delay=max_delay))
+
+    @classmethod
+    def jittered_backoff(cls, retries=10, delay=3, max_delay=60):
+        """
+        Retry calling the Cloud decorated function using a jittered backoff
+        strategy. More on this strategy here:
+
+        https://www.awsarchitectureblog.com/2015/03/backoff.html
+
+        Kwargs:
+            retries (int): Number of times to retry a failed request before giving up
+                default=10
+            delay (int): Initial delay between retries in seconds
+                default=3
+            max_delay (int): maximum amount of time to wait between retries.
+                default=60
+        """
+        return cls._backoff(_full_jitter_backoff(
+            retries=retries, delay=delay, max_delay=max_delay))
+
+    @classmethod
+    def backoff(cls, tries=10, delay=3, backoff=1.1):
```

Comment:
This is an `exponential_backoff` with a different exponent. If we're going to replace `backoff` with `exponential_backoff`, it should have equivalent defaults (and remove the duplicated code)